src/Components/Input/index.js

```diff
@@ -3,12 +3,27 @@ import React, { Component } from 'react';
 import './index.css';
 
 export default class Input extends Component {
-    render() {
+    withLabel() {
         return (
             <div className="Input">
                 <label className="Input-label">{this.props.label}</label>
                 <input className="Input-type" type={this.props.type} />
             </div>
         )
     }
+
+    withoutLabel() {
```

Comment:
I think you should rename these functions names to `renderWithLabel` and `renderWithoutLabel`. It's a convention on the react community to help differentiate functions that return `html` from the ones that take care of behaviour/business logic.

Reply:
gotcha!